src/static/js/modules/header.js

```diff
@@ -1,9 +1,11 @@
 const hamburgerMenu = document.querySelector('.burger-menu');
 const close = document.querySelector('.button-close');
+const logos = document.querySelectorAll('.mor-logo');
```

Comment:
Don't do this, this should be an anchor wrapping the <img>. check my comment above in the HTML.